.github/workflows/publish-python-wrapper-client.yaml

```diff
@@ -32,4 +33,34 @@ jobs:
         packages-dir: clients/python-wrapper/dist/
         verbose: true
 
-# TODO: Publish docs once we decide on documentation process
+    - name: Extract docs version
+      working-directory: ./clients/python-wrapper/
+      shell: bash
+      run: |
+          echo "tag=$(python setup.py --version)" >> $GITHUB_OUTPUT
+      id: docver
+
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.9
+
+    - name: Generate and Build Documentation
+      working-directory: ./clients/python-wrapper/
+      run: |
+        echo ${{ steps.docver.outputs.tag }}
+        pip install -r requirements.txt 
+        sphinx-apidoc -o docs lakefs sphinx-apidoc --full -A 'Treeverse' -e
+        sphinx-build -b html docs docs/_build/${{ steps.docver.outputs.tag }}
```

Comment:
The sphinx quickstart also generates a Makefile under docs that used to build the docs. We should just `make html` at this step to have it working inside and outside the workflow.